fiware-region-sanity-tests/commons/constants.py

```diff
@@ -38,6 +38,7 @@
 PROPERTIES_CONFIG_CRED_TENANT_NAME = "tenant_name"
 PROPERTIES_CONFIG_CRED_USER_DOMAIN_NAME = "user_domain_name"
 PROPERTIES_CONFIG_CRED_USER = "user"
+PROPERTIES_CONFIG_CRED_USER_ID = "user_id"
 PROPERTIES_CONFIG_CRED_PASS = "password"
 PROPERTIES_CONFIG_TEST = "test_configuration"
```

Comment:
To avoid confusion, and align with OpenStack conventions, please refactor:
* `PROPERTIES_CONFIG_CRED_USER = "user"` --> `PROPERTIES_CONFIG_CRED_USERNAME = "username"`
* `PROPERTIES_CONFIG_CRED_PASS` --> `PROPERTIES_CONFIG_CRED_PASSWORD`